frontend/src/constants/panelTypes.ts

```diff
@@ -9,10 +12,27 @@ export const PANEL_TYPES_COMPONENT_MAP = {
 	[PANEL_TYPES.VALUE]: GridValueComponent,
 	[PANEL_TYPES.TABLE]: GridTableComponent,
 	[PANEL_TYPES.TRACE]: null,
-	[PANEL_TYPES.LIST]: null,
+	[PANEL_TYPES.LIST]: LogsPanelComponent,
```

Comment:
Why LogsPanelComponent ?

Reply:
If someone clicks to list view, by default LogsPanelComponent should render. Then we can switch to traces or logs.